docs/guides/component-testing/component-framework-configuration.mdx

```diff
@@ -45,26 +70,34 @@ resolves to a `devServer` instance.
 
 :::
 
-## Custom Index File
-
-By default, Cypress renders your components into an HTML file located at
-`cypress/support/component-index.html`.
+Any requests triggered during a test using the `devServerPublicPathRoute` as
+defined in the `cypressConfig` will be forwarded to your server. The
```

Comment:
Might be worth including you can access the user's HTML file via `options.cypressConfig.indexHtmlFile` in your custom integration.